packages/vscode-extension/src/commonlib/m365Login.ts

```diff
@@ -71,6 +71,8 @@ export class M365Login extends BasicLogin implements M365TokenProvider {
   private static instance: M365Login;
   private static codeFlowInstance: CodeFlowLogin;
 
+  private static cachedJsonObject: Record<string, unknown>;
```

Comment:
Is it possible to just reuse `CodeFlowLogin.account`? That's already the cached account.

Reply:
Nice idea! Updated to use `CodeFlowLogin.account`. Thanks.